<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="dataSourceStorageLocal" created-in="PY-213.7172.26">
    <data-source name="data" uuid="e8a55d5f-b4c2-413d-aecf-a83aac93a8bf">
      <database-info product="" version="" jdbc-version="" driver-name="" driver-version="" dbms="SQLITE" exact-version="0" />
      <auth-provider>no-auth</auth-provider>
      <schema-mapping />
    </data-source>
    <data-source name="data_namaz" uuid="98275a18-7bb5-4ff1-98d1-1a40a684e05e">
      <database-info product="SQLite" version="3.36.0" jdbc-version="2.1" driver-name="SQLite JDBC" driver-version="3.36.0.3" dbms="SQLITE" exact-version="3.36.0" exact-driver-version="3.36">
        <identifier-quote-string>&quot;</identifier-quote-string>
      </database-info>
      <case-sensitivity plain-identifiers="mixed" quoted-identifiers="mixed" />
      <auth-provider>no-auth</auth-provider>
      <schema-mapping>
        <introspection-scope>
          <node kind="schema" qname="main" />
        </introspection-scope>
      </schema-mapping>
    </data-source>
  </component>
</project>